single_tech_samples/synapseanalytics/sample2_synapseserverless/README.md

```diff
@@ -83,6 +169,57 @@ The diagram below illustrates the step-by-step process in order to successfully
 
 ![CLS](./images/CLS.jpg)
 
+### Applying ACLs to the datalake (using a Spark pool)
+
+Once the ingestion is finished, a process that applies the ACL list defined in the configuration file in the datalake is triggered.
+The main advantage of this process is that runs as an independent module. It can run after the ingestion within the same pipeline, but it can also be ran independently of the ingestion process if necessary.
+
+In order to apply the ACLs, the azure-storage-file-datalake library was used and a Key Vault was included in the deployment to deal with the storage secrets properly.
+
+Nevertheless, importing the library mentioned above might take a considerably amount of time from within Synapse.
+In the table below the explored options with pros and cons are summarized:
+
+| Method | Command used/Steps needed | Pros | Cons |
+|-|-|-|-|
+| Import library directly in the notebook | pip install azure-storage-file-datalake | Faster method to load the library (seconds after the Spark pools starts) | |
+| Import requirements.txt and update Spark pool | az synapse spark pool update with --library-requirements requirements.txt | 1) Centralized execution of the requirements | Takes 20 minutes to run this step |
+| | | 2) Runs one time at the deployment time | |
```

Comment:
Seems there is not really an obvious way to get rid of the empty cells because merge is not working and html tags are not allowed. I'm skipping the line for now and adding the second point to the previous cells, but also the <br> tag is not allowed.